src/controllers/cron/update_local_smartapi.js

```diff
@@ -223,6 +223,7 @@ const getAPIOverrides = async (data) => {
         };
         const index = overrides.conf.only_overrides ? -1 : data.hits.findIndex(hit => hit._id === id);
         if (index === -1) {
+            debug(`WARN: Overridden API ID ${id} not recognized, appending as new API hit.`);
```

Comment:
@tokebe you may use the "formal" `warning` package here (https://www.npmjs.com/package/warning)? Potentially we can use this package in other places for warning msgs. Although I don't know what actual difference it will be at the output. Not a big deal of course.

Reply:
The main change I'm seeing is that we can't have the fancy colorful `bte:biothings-explorer-trapi:cron` label at the beginning of the log message, otherwise it works as expected. If it's preferred to have that, I can change back to `debug`, otherwise my next commit changes to the warning package for this warning.

Reply:
I see, then probably not worth using a different logger for warning msg, so let's stick with `debug`.

Reply:
@tokebe can you please remove the use of `warning` package, then I will merge this PR?